extensions/amp-stream-gallery/amp-stream-gallery.md

```diff
@@ -59,7 +59,179 @@ navigational arrows to go forward or backwards by a given number of items.
+      background: rebeccapurple;
+      background: linear-gradient(90deg, rebeccapurple 50%, mediumpurple 90%, mediumslateblue 100%);
+    }
+    .cyan-gradient {
+      background: darkcyan;
+      background: linear-gradient(90deg, darkcyan 50%, lightcyan 90%, white 100%);
+    }
+  </style>
+</head>
+<amp-stream-gallery id="my-carousel" max-visible-count="3">
+  <div class="red-gradient"></div>
+  <div class="blue-gradient"></div>
+  <div class="green-gradient"></div>
+
+  <div class="pink-gradient"></div>
+  <div class="yellow-gradient"></div>
+  <div class="orange-gradient"></div>
+
+  <div class="seafoam-gradient"></div>
+  <div class="purple-gradient"></div>
+  <div class="cyan-gradient"></div>
+</amp-stream-gallery>
+<div class="buttons" style="margin-top: 8px;">
+  <button id='prev-button'>Go to previous page of slides</button>
+  <button id='next-button'>Go to next page of slides</button>
+  <button id='go-to-button'>Go to slide with green gradient</button>
+</div>
+<script>
+  (async () => {
+    const carousel = document.querySelector('#my-carousel');
+    await customElements.whenDefined('amp-stream-gallery');
+    const api = await carousel.getApi();
+    // programatically advance to next slide
+    api.next();
+    // set up button actions
+    document.querySelector('#prev-button').onclick = () => api.prev();
+    document.querySelector('#next-button').onclick = () => api.next();
+    document.querySelector('#go-to-button').onclick = () => api.goToSlide(2);
+  })();
+</script>
+```
+
+[/example]
+
+#### Interactivity and API usage
+
+Bento enabled components in standalone use are highly interactive through their API. In Bento standalone use, the element's API replaces AMP Actions and events and [`amp-bind`](https://amp.dev/documentation/components/amp-bind/?format=websites).
+
+The `amp-stream-gallery` component API is accessible by including the following script tag in your document:
+
+```
+await customElements.whenDefined('amp-stream-gallery');
+const api = await carousel.getApi();
+```
+
+##### Actions
+
+The `amp-stream-gallery` API allows you to perform the following actions:
+
+**next()**
+Moves the carousel forwards by `advance-count` slides.
+
+```
+api.next();
+```
+
+**prev()**
+Moves the carousel backwards by `advance-count` slides.
+
+```
+api.prev();
+```
+
+**goToSlide(index: number)**
+Moves the carousel to the slide specified by the `index` argument.
+Note: `index` will be normalized to a number greater than or equal to `0` and less than the number of slides given.
+
+```
+api.goToSlide(0); // Advance to first slide.
+api.goToSlide(length - 1); // Advance to last slide.
+```
+
+##### Events
+
+The `amp-stream-gallery` API allows you to register and respond to the following events:
+
+**slideChange**
+
+This event is triggered when the index displayed by the carousel has changed.
+The new index is available via `event.data.index`.
+
+```
+carousel.addEventListener('slideChange', (e) => console.log(e.data.index))
```

Comment:
So, is it "slideChange" or "slidechange"? IMHO this should be all lower-case in all cases.

Reply:
Keeping this as `slideChange` for now based on discussion in #32194